Reqnroll.Generator/Generation/UnitTestFeatureGenerator.cs

```diff
@@ -385,6 +385,19 @@ private void SetupTestCleanupMethod(TestClassGenerationContext generationContext
 
             var testRunnerField = _scenarioPartHelper.GetTestRunnerExpression();
 
+            // Guard when calling AfterScenario when TestRunner is already released (errors or edge cases in a supported test framework)
+            // if ((testRunner == null))
```

Comment:
what's the idea with this commented out code?

Reply:
In the generator code, the commented lines demonstrate the code that will get generated by the CodeDom code that follows the comment.

Reply:
Maybe we could prefix the comments with: `generates:`